src/core/ext/filters/client_channel/client_channel.cc

```diff
@@ -310,6 +310,75 @@ class CallData {
  private:
   class QueuedPickCanceller;
 
+  class Metadata : public LoadBalancingPolicy::MetadataInterface {
+   public:
+    class Iterator : public MetadataInterface::IteratorInterface {
+     public:
+      explicit Iterator(grpc_linked_mdelem* linked_mdelem)
+          : linked_mdelem_(linked_mdelem) {}
+
+      grpc_linked_mdelem* linked_mdelem() const { return linked_mdelem_; }
+
+      virtual IteratorInterface& operator=(
+          const IteratorInterface& other) override {
+        const Iterator* it = static_cast<const Iterator*>(&other);
+        linked_mdelem_ = it->linked_mdelem_;
+        return *this;
+      }
+
+      virtual IteratorInterface& operator++() override {
```

Comment:
nit: `virtual` is unnecessary when `override` is added. can you please remove them here and elsewhere.

Reply:
Thanks!